packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -254,11 +255,84 @@ The requested scope is invalid:
+| scope                   | A list of scopes that the client wants included in the access token. See [scopes](#access-token-scopes-and-claims).                                                                                                                                                                                                 | String |
+
+#### Response properties
+
+Based on the scopes requested. Generally speaking, the scopes specified in a request are included in the access token in the response.
+
+| Property                  | Description                                                                                    | Type    |
+| :------------------------ | :--------------------------------------------------------------------------------------------- | :------ |
+| verification_uri          | The verification URI that the resource owner must visit to authenticate.                            | String  |
+| verification_uri_complete | The complete verification URI that the resource owner can alternatively visit to authenticate.      | String  |
+| device_code               | The device code to be kept hidden from the resource owner.                                     | String  |
+| user_code                 | The user code to be shown to the resource owner.                                               | String  |
+| interval                  | The interval time the client should wait until it can poll the /token endpoint in seconds.     | Integer |
+| expires_in                | The expiration time of the device code and user code in seconds.                               | Integer |
+
+#### List of errors
+
+| Error Id                 | Details                                                                                                                                                                                                    |
+| :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| invalid_client           | The specified `client_id` isn't found.                                                                                                                                                                      |
+| invalid_request          | The request structure is invalid. For example, the basic authentication header is malformed, both header and form parameters are used for authentication, no authentication information is provided, or the request contains duplicate parameters. |
```

Comment:
The request is missing a necessary parameter, the parameter has an invalid value, or the request contains duplicate parameters.